packages/squiggle-lang/__tests__/TS/TestHelpers.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  run,
+  Distribution,
+  squiggleExpression,
+  errorValueToString,
+  errorValue,
+  result,
+} from "../../src/js/index";
+
+export function testRun(x: string): any {
```

Comment:
I'll fix this one for you if you want

Reply:
How so? A resolution to the return type of `testRun` would lead to a few refactors of tests around here (You can tell sorta code I wrote before I gave up on avoiding `any` vs. after. all day yesterday I _really_ wanted `testRun(x).value.value` and it wouldn't let me have it till I marked it `any` lol. Yesterday was the first real typescript day of my life, so forgive me!)